docs/fiddles/ipc/pattern-2/main.js

```diff
@@ -1,4 +1,4 @@
-const {app, BrowserWindow, ipcMain} = require('electron')
+const {app, BrowserWindow, ipcMain,dialog} = require('electron')
```

Comment:
```suggestion
const {app, BrowserWindow, dialog, ipcMain} = require('electron')
```
spacing adjustment and alphabetization. good catch, ty.